Directory.Build.props

```diff
@@ -13,14 +13,14 @@
     <DefaultLanguage>en-US</DefaultLanguage>
 		
 		<!--<TargetFrameworksOverride>xamarinios10</TargetFrameworksOverride>-->
-		<TargetFrameworksOverride>uap10.0.16299;MonoAndroid80;xamarinios10;netstandard2.0</TargetFrameworksOverride>
+		<TargetFrameworksOverride>uap10.0.17763;MonoAndroid80;xamarinios10;netstandard2.0</TargetFrameworksOverride>
```

Comment:
The main repo is staying at 16299 for now, is that going to cause any problems here having a different target for Uno?

Reply:
Not really at this point. For now, it creates dependency issues with 16.4, not sure where that comes from, could be caused by this:

https://github.com/unoplatform/uno/blob/83ec656dcf81992e58693b8e6a1d6ea59810f339/build/Uno.UI.nuspec#L223

I'll add 16299 to be safe. In any case at this moment, it's better to use the main WCT packages for UAP, not the uno ones.